### DISCUSSION TITLE
Divergent Universe/ Community

### DISCUSSION THREAD

**Discussion by cleo-dev** [LmeSzinc/StarRailCopilot#693] [Q&A] [answered]
Well, I actually have a few questions.
First of all, thanks for the project! I think it's amazing. You guys are great.
Well, I wanted to know if there are any plans for the divergent universe to become automated, like the simulated universe. It would be interesting to at least automatically farm the Synchronicity Level levels. In addition to this doubt, I also have a question about the community... I'm Brazilian, I don't speak Chinese and my English is rusty but it's enough... I've noticed that most of the content on github is in Chinese and I'd like to interact with people who use SRC to give me some tips on how I can modify it to make it better. Is there something like a reddit community or discord?

Anyway, thanks again, you guys are amazing!

**Comment by LmeSzinc** [comment, marked as answer]
Yes, there is plan for divergent universe. Readme has a language switch at the very beginning and the discord link is in it.
